data/mods/ssb/random-teams.ts

```diff
@@ -34,6 +34,12 @@ export const ssbSets: SSBSets = {
 	// Nature needs to be a valid nature with the first letter capitalized ex: Modest
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+	Flare: {
+		species: 'Weavile', ability: 'Permafrost Armor', item: 'Life Orb', gender: 'M',
```

Comment:
``FlareToday at 5:00 PM
Ah minor fix on my Pokémon please. I noticed in the code it says the gender is "m" but I stated that it should be genderless``
Gender should be N